gno.land/pkg/integration/testdata/gnokey_qpaths.txtar

```diff
@@ -0,0 +1,121 @@
+# aaa namespace
+loadpkg gno.land/r/aaa/bbb/ccc $WORK
+loadpkg gno.land/r/aaa/ccc/ddd $WORK
+loadpkg gno.land/r/aaa $WORK
+
+# bbb namespace
+loadpkg gno.land/r/bbb/ccc $WORK
+loadpkg gno.land/p/bbb/ccc/ppp $WORK
+loadpkg gno.land/r/bbb/aaa $WORK
+loadpkg gno.land/r/bbb0 $WORK
+
+gnoland start
+
+# query path for prefix
+gnokey query vm/qpaths --data "gno.land/r/aaa"
+cmp stdout aaa-qpaths.stdout.golden
+cmp stderr aaa-qpaths.stderr.golden
+
+gnokey query vm/qpaths --data "gno.land/r/bbb"
+cmp stdout bbb-qpaths.stdout.golden
+cmp stderr bbb-qpaths.stderr.golden
```

Comment:
`! stderr .` - more direct, no?

Reply:
It's a no-brainer copy/paste, and I also feel it's easier to spot issues with later potential modifications. If any errors show up, they will be marked as a diff.

Reply:
Feel free to resolve this, but I think this adds complexity reading. This "cmp stderr" is simply saying "ensure the stderr is empty" - maybe as a compromise, you can make an `empty` file and just do `cmp stderr empty`?